config/settings.yml

```diff
@@ -1023,6 +1023,7 @@
   :interval: 60
   :memcache_server: 127.0.0.1:11211
   :memcache_server_opts: "-l 127.0.0.1"
```

Comment:
Ok, so I think I understand it based on what you've said.  So, dalli's `error_when_over_max_size` doesn't actually toggle whether it tries to store it in memcached, only whether it raises an exception or simply does nothing.  Basically, if you exceed `value_max_bytes`, dalli will not try to store the data in session.

**My proposal**:
Let's pass a large value for `value_max_bytes` to dalli and don't expose it.  Similar to what you have NOW but without getting the value from configuration.  

**Before**:
```
:cache => Dalli::Client.new(Settings.session.memcache_server, :namespace => "MIQ:VMDB")
```

**After**:

```
:cache => Dalli::Client.new(Settings.session.memcache_server, :namespace => "MIQ:VMDB", :value_max_bytes => 10.megabytes)
```

Maybe move the 10.megabytes to a constant at the top of session_store.rb.  If 10 isn't enough, choose a higher number.

Basically, we'll only be limited by the setting we pass to `memcached`.  That way, you only have to configure the setting we ALREADY expose.

Then we change our default in settings.yml:

`memcache_server_opts: "-l 127.0.0.1"`

To:

`memcache_server_opts: "-l 127.0.0.1 -I 1M"`

So, then you only have to change 1M to 3M or whatever you want it to be.
In this way, we keep everything as they were before, but make it fairly simple to allow large sessions by just changing 1M to 3M or whatever you want and saving.  You'll still need to restart as I don't think we bounce memcached when these settings change.

cc @nenioscio @bdunne @Fryguy @gmcculloug @dclarizio (what do you think?)